contracts/MechMarketplace.sol

```diff
@@ -415,17 +486,9 @@ contract MechMarketplace is IErrorsMarketplace {
         }
         _locked = 2;
 
-        // Check agent mech and get its mech service multisig
+        // Check delivery mech and get its service multisig
         address mechServiceMultisig = checkMech(msg.sender, deliveryMechStakingInstance, deliveryMechServiceId);
 
-        // Get the staked service info for the mech
-        IStaking.ServiceInfo memory serviceInfo =
-            IStaking(deliveryMechStakingInstance).getServiceInfo(deliveryMechServiceId);
-        // Check that staked service multisig is the priority mech mech service
-        if (!IMech(msg.sender).isOperator(serviceInfo.multisig)) {
-            revert UnauthorizedAccount(msg.sender);
-        }
```

Comment:
why removed?

Reply:
Exactly same check is performed in a function above: `checkMech()`